lib/DDG/Spice/Symbolab.pm

```diff
@@ -7,9 +7,9 @@ spice is_cached => 1;
 spice to => 'https://www.symbolab.com/ddg?query=$1';
 spice wrap_jsonp_callback => 1;
 
-triggers startend => "calculate", "solve", "compute";
-handle remainder => sub {
-    if ($_ =~ /[\^\*\=\+\-\/\\]/ or $_ =~ /\b(sin|cos|tan|cot|csc|sec|ln|log|sqrt|integral)/i) {
+triggers startend => "calculate", "solve", "compute", "integral", "integrate", "integration", "antiderivative", "derivative", "derive", "differentiate", "derivation";
+handle query_lc => sub {
+    if ($_ =~ /[\^\*\=\+\-\/\\]/ or $_ =~ /\b(sin|cos|tan|cot|csc|sec|ln|log|sqrt|integr|deriv|diff)/i) {
```

Comment:
@levaly there is a problem here: your regex is matching against the query for `integr` and the trigger is `integration`.

This means every query that contains the word "integration", "integrated", or ""integrate", etc seems like a valid query and triggers this IA

We need to fix this and likely add a second check for the presence of numbers `\d+` in the query in certain cases